doc/api/crypto.md

```diff
@@ -2383,7 +2383,7 @@ An array of supported digest functions can be retrieved using
 <!-- YAML
 added: v0.11.14
 changes:
-  - version: REPLACEME
+  - version: v12.10.0
```

Comment:
Shouldn't all of these `REPLACEME` replacements be `v12.11.0`?

Reply:
> Shouldn't all of these `REPLACEME` replacements be `v12.11.0`?

Every time I see a comment from @richardlau, my reaction is always "Wow, I am glad that comment happened."

Reply:
Resolved.